### ISSUE TITLE
`The internal SQL parser failed to parse the SQL statement` for a query with `ANY`

### ISSUE THREAD

**Issue by lenguyenthanh** [akheron/sqltyper#30]
Below are my simpler version of my query and the error message.
If you need I'll try to create a sample data and query.

## Query

```
select * 
from table1
where not table1.id = any(
	select table1.id
	from table2
	where table2.id <> '1'
)```

## Error Message

```
WARNING: The internal SQL parser failed to parse the SQL statement.

Parse error: Expect symbol `)` at (7:2)

    1| select * 
    2| from table1
    6| where not table1.id = any(
    7|  select table1.id
        ^

```

P/S Thanks for your great library. I'll try not only report issues but also contribute to this project if I can.

**Comment by lenguyenthanh** [comment]
It also doesn't work with keyword `using`.

**Comment by akheron** [comment]
Yeah, support for `any/some/all` does not yet exist in the parser.

In your case, you could use `in` as a workaround, because it’s equal to `= any`.

**Comment by akheron** [comment]
Could you open a separate issue for `using`?

**Comment by lenguyenthanh** [comment]
Yes, I'll create new issue for using. Do we have a list of supported syntax? I'll be really great if we have that.

**Comment by akheron** [comment]
Unfortunately no. The goal is to eventually support all PostgreSQL syntax.

**Comment by lenguyenthanh** [comment]
Understand, I'll try to work on one of the syntax this weekend.

**Comment by akheron** [comment]
Nice! Let me know if you need help.
